.github/workflows/build.yml

```
@@ -369,6 +369,57 @@ jobs:
+      - name: Load database image
+        id: load_image
+        run: |
+          docker load --input /tmp/falkordb-${{ matrix.platform.suffix }}.tar
+
+      - name: Clone browser repository
+        run: |
+            LATEST_TAG=$(git ls-remote --tags --sort=-v:refname https://github.com/FalkorDB/falkordb-browser.git | awk -F'/' '{print $NF}' | grep -E '^v[0-9]+(\.[0-9]+)*$' | head -n1)
+            git clone --depth=1 --branch $LATEST_TAG --single-branch https://github.com/FalkorDB/falkordb-browser.git /tmp/browser
+            cd /tmp/browser
+            git describe --tags
+                
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)